.agignore

```diff
@@ -0,0 +1 @@
+app/assets/javascripts/metamaps.secret.bundle.js
```

Comment:
i use the "silver searcher" to recursively search for strings in the codebase. I kept getting results in this file, so I ignored it. 

I could remove this file if metamaps.secret.bundle.js were in the "webpacked" directory, which is gitignored. But I guess the current approach has this file checked into git (which is probably bad). Ideally we would have this as a second bundle generated by webpack.config.js